src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/common/SupportedAppointmentTypes.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.common
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.config.Feature
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.config.FeatureSwitches
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.BookingHistoryAppointment
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.PrisonAppointment
+
+@Component
+class SupportedAppointmentTypes(private val featureSwitches: FeatureSwitches) {
```

Comment:
So I do have a question on this.  Instead of a feature toggle could this be based on reference data in the DB?

e.g. VIDEO_LINK_APPOINTMENT_TYPES and then enabled accordingly based on environment.

Reply:
Feature toggle is probably the simplest? As you have it here.
It's quite a big deal to add more appointment types, and not something we'd turn on just by adding an extra type in the DB?